tests/admin_docs/views.py

```diff
@@ -27,3 +27,14 @@ class CompanyView(View):
 
     def get(self, request):
         return HttpResponse()
+
+
+class FamilyView(View):
```

Comment:
We try to avoid new views/classes/models in tests if we can, I think doing the model docstring is enough so I would remove this here (and from urls and the test)